.github/workflows/outdated-integrations.yml

```diff
@@ -0,0 +1,35 @@
+name: Outdated Integrations
+on:
+  workflow_dispatch: # manually
+  push:
+    branches:
+      - quinna.halim/update-outdated-integrations-workflow # TODO: remove, for testing
+
+permissions: read-all
+
+concurrency:
+  # Automatically cancel previous runs if a new one is triggered to conserve resources.
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: 'Outdated Integrations'
+    runs-on: ubuntu-latest
+    env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+      
+      - run: go run latest_major_version.go > latest_majors.txt
+
+      - name: Create Pull Request
+        id: pr
+        uses: peter-evans/create-pull-request@v6
```

Comment:
From what I see for now, you are only outputting a text file with `New latest major version`, which I think it doesn't make much sense to be committed to the repo.

What about sending a slack notification for now? We can work on some sort of code-change actionable in the future.